kernel_gateway/notebook_http/request_utils.py

```diff
@@ -12,17 +12,20 @@
 APPLICATION_JSON = 'application/json'
 TEXT_PLAIN = 'text/plain'
 
-def format_request(bundle):
+def format_request(bundle, kernel_language='no_spec'):
     """Creates an assignment statement of bundle JSON-encoded to a variable
-    named `REQUEST`.
+    named `REQUEST` by default or kernel_language specific.
 
     Returns
     -------
     str
         `REQUEST = "<json-encoded expression>"`
```

Comment:
This line of the doc string is no longer accurate after the change, right?

Reply:
I think it also accurate, too. It'll keep the variable as `REQUEST` for those languages works well and a similar variable name for those kernel_language specific ones. I think the description is consistent to following codes logically.

Reply:
Line 22 says that `REQUEST =` is returned, unconditionally.
Line 17 says that the return value can be different, depending on the kernel language.
That's a contradiction.

Reply:
Yes, Line 22 should be updated.